src/jackdaw/streams/extras.clj

```diff
@@ -39,7 +38,7 @@
                              ^String storeName
                              ^long startingOffset
                              ^long endingOffset]
-       (swap! restore-tracker assoc storeName (inst/now))
+       (swap! restore-tracker assoc storeName (java.time.Instant/now))
```

Comment:
How about importing `Instant` and `Duration` at the top? :) 

🙇  Thanks for the contribution

Reply:
sure thing